subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -583,6 +653,19 @@ void radio_switch_complete_and_tx(uint8_t phy_rx, uint8_t flags_rx, uint8_t phy_
 #endif /* !CONFIG_BT_CTLR_TIFS_HW */
 }
 
+void radio_switch_complete_and_pback(uint8_t phy, uint8_t flags)
```

Comment:
```suggestion
void radio_switch_complete_and_b2b_tx(uint8_t phy, uint8_t flags)
```

i.e. if using the term back-to-back is ok? That is what is happening in reality. I am ok with piggyback/pback.

Reply:
back2back sounds better, will use it